LocalIdentity/PathOfBuilding
# Add support for Fleshcrafter minion life conversion

This requires a change to compute minion resistances before life. Player resistances must be computed after auras, so the calc pipeline isn't equivalent for the two actors. It looks like there's already separate handling for minions, so the main change is that section.

Also fixes #6149.

### Description of the problem being solved:
Fleshcrafter minion life conversion is currently unsupported
### Steps taken to verify a working solution:
- Use Fleshcrafter in build
- Check life/ES values after changing minion chaos resistance on gear/tree

### Link to a build that showcases this PR:
https://pobb.in/iUOglRJrlnB0
### Before screenshot:
No minion life converted to ES
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/144673064/3835657a-b940-4c99-909d-e0204cb0f8ad)

### After screenshot:
Now minion shows life converted to ES:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/144673064/3cb7a451-d9eb-4a31-8981-b97a67ea06e9)
If the minion has over 50% chaos res, life shows as 0:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/144673064/f5de4656-01f0-48e3-9e40-2225c40dccc9)
Added warning for current active minion if all life is converted:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/144673064/c45fc6fc-4868-4706-9687-040633c4c47f)
